topics/galaxy-interface/tutorials/upload-rules-advanced/tutorial.md

```diff
@@ -93,7 +93,7 @@ We will use this information to build a collection of FASTA files from our query
 >      ```
 >
 >    - Click `Build`
->   ![screenshot](../../images/rules/rules_example_4_2_initial_rules.png)
+>   ![screenshot](http://galaxy-tests.s3.amazonaws.com/galaxy-gtn-screenshots/local/rules_example_4_2_initial_rules.png)
```

Comment:
The GTN is served with HTTPS only, and some browsers will throw an error for mixed content like this, I think we might need to find a solution there. (It could be as simple as proxying it, like we do in some other cases https://github.com/galaxyproject/infrastructure-playbook.)

Additionally it's unfortunate that this will now require an internet connection just to view the tutorial, but I definitely understand the ease of doing it this way. Maybe we can find some alternative solution in the future there.